features/adb-openshift.feature

```diff
@@ -57,6 +57,12 @@ Feature: Command output from various OpenShift related commands in ADB
     Then the exit status should be 0
     And the binary "oc" should be installed
 
+    # '../vagrant.d' maps to ENV['VAGRANT_HOME']
+    When I run `bundle exec vagrant service-manager install-cli openshift --cli-version 1.3.0 --path ../vagrant.d/oc`
```

Comment:
Here is what I came up with. The issue is really that Aruba does not execute the command in backticks directly and hence interpolation will take place. What's about:

        When I evaluate and run `bundle exec vagrant service-manager install-cli docker --path #{ENV['VAGRANT_HOME']}/docker`
        Then the exit status should be 0
        And the binary should be installed in path "#{ENV['VAGRANT_HOME']}/docker"

To make this work we need a new step defintion:

    # Per default the built-in Aruba 'When I run' will not evaluate/interpolate the string.
    # We create a custom step definition to work around this
    When(/^I evaluate and run `([^`]*)`$/)do |cmd|
        cmd = eval('"' + cmd + '"')
       cmd = sanitize_text(cmd)
       run_simple(cmd, :fail_on_error => false)
    end

Note, I also changed the way ' And the binary should be installed in path' is implemented:

    Then(/^the binary should be installed in path "([^"]*)"$/) do |path|
      binary_path = eval('"' + path + '"')
      expect(File.file?(binary_path)).to eq(true)
      expect(File.executable?(binary_path)).to eq(true)
    end

With this in place one can consistently use '#{ENV['VAGRANT_HOME']}' (or any other variable really). I think this is a "cleaner" solution. Everything is explicit. WDYT?